patches/api/0495-Add-new-TickRateChangedEvent.patch

```diff
@@ -0,0 +1,82 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Axionize <154778082+Axionize@users.noreply.github.com>
+Date: Tue, 8 Oct 2024 20:38:01 -0400
+Subject: [PATCH] Add new TickRateChangedEvent
+
+
+diff --git a/src/main/java/org/bukkit/event/server/TickRateChangeEvent.java b/src/main/java/org/bukkit/event/server/TickRateChangeEvent.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..279e471f00495cc08ed2297efb21fb801cd43851
+--- /dev/null
++++ b/src/main/java/org/bukkit/event/server/TickRateChangeEvent.java
```

Comment:
Paper events go in the paper package

Reply:
I was thinking about upstreaming this into Spigot later

Reply:
if that's the case, then this PR is inherently blocked as it makes little sense for us to finalise the review and such of this into a state we're happy to merge it if we're going to potentially have to deal with breaking ABI changes from upstream

Reply:
Which will inherently be the case as spigot does not expose the CommandSourceStack this event now provides.
Would be great if you can come to a decision whether or not you want to have this event in spigot-api soonish then. As cat mentioned, if this is going upstream, there is 0 point in this PR receiving reviews on paper and we can close it instead.

Reply:
Any update on this? I'd really like to either merge or close this depending on your decision regarding moving this to upstream.

Reply:
I've decided to leave it in paper. Are there any other changes you'd like me to make?

Reply:
Well as cat stated, the event should be in the paper event package